README.md

```diff
@@ -1,2 +1,29 @@
 # comp110-worksheet-B
-Base repository for COMP110 worksheet B
+
+Attempt Number = 4
+
+x = Correct Letters  
+
+Generate Random Words
+
+OUTPUT " ROBCO INDUSTRIES (TM) TERMALINK PROTOCOL" 
+OUTPUT " ENTER PASSWORD NOW" 
+
+OUTPUT " 4 Attempts Left" 
+
+INPUT User Word Guess
+
+If INPUT Incorrect
+
+Then OUTPUT x
```

Comment:
Use indentation to make it clear which code is nested inside your if statements.